package/gluon-ebtables-filter-multicast/luasrc/lib/gluon/ebtables/110-mcast-allow-respondd

```diff
@@ -0,0 +1,2 @@
+rule 'MULTICAST_OUT -p IPv6 --ip6-protocol udp --ip6-destination-port 1001 --ip6-dst ff02::2:1001 -j RETURN'
```

Comment:
The first rule doesn't seem useful - current Gluon nodes don't bind to `ff02::2:1001` over the mesh interface anyways.

Reply:
Fine for me, FFHL has converged to ff05::2:1001. Less legacy to carry along in Gluon without that rule.